### ISSUE TITLE
Media Play Pause icon never disappear

### ISSUE THREAD

**Issue by zetaloop** [mulaRahul/keyviz#264]
**Describe the bug**
The `media play pause` icon shows up and never disappear after click play/pause button on my headphone.

**Version**: v2.0.0-alpha2

**To Reproduce**
A. My bluetooth headphone
1. Click play/pause button on my headphone
2. The play/pause icon shows up and never disappear

B. Use Win + L instead (I guess it's the same issue)
1. Click <kbd>Win</kbd> + <kbd>L</kbd>
2. Unlock the screen
3. The icons are still there

**Expected behavior**
The icon should disappear after click.
Or at least it can disappear after a certain timeout, if we just can't catch the key release event.

**Screenshots**
![pause, win, L](https://github.com/mulaRahul/keyviz/assets/36418285/057f3ab7-9566-4870-b87f-d65ceed2dc59)

**Desktop (please complete the following information):**
 - Windows 11 Build 22635.3570

**Additional context**
 - Headphone: OnePlus Buds Pro2

**Comment by zetaloop** [comment]
Maybe we could add a timeout to force the icon to disappear. For example, after a single press, the key is displayed for up to 2 seconds, and then slowly fades out.
This can alleviate the problem of the key not disappearing, and has little impact on real long presses.

**Comment by mulaRahul** [comment]
Yeah, that's a know bug. Will figure out a way around the same in the next release.
